package.json

```diff
@@ -27,12 +27,13 @@
   },
   "scripts": {
     "ng": "ng",
+    "clean:workspace": "del ./**/dist && yarn clean:modules",
     "clean:modules": "del ./**/node_modules",
     "clean": "del ./dist && yarn run schematics:clean",
     "start": "yarn angular:dev:start",
     "build": "npm-run-all build:ci build:website build:storybook:angular build:storybook:core angular:dev:build",
     "build:ci": "npm-run-all clean format lint build:libs test",
-    "build:libs": "npm-run-all -p ui:build icons:build core:build -s angular:build schematics:build react:build",
+    "build:libs": "npm-run-all -p ui:build icons:build core:build -s link:packages angular:build schematics:build react:build link:packages",
```

Comment:
The first call is needed here to make sure when the `@clr/core` is built - it can be used inside other packages that depend on it and the last one is needed to `link` the rest of the packages cause in the first run they weren't ready to link

I believe also that this `link` actually can be skipped and when we need to interact with local builds we are going to run the task described on line `49` to make the local package linking.

Reply:
A possible future improvement but not needed for this PR is to have the link script take an argument and each project can call it directly so each project would build then link itself. That might add a bit more time than linking them all at once but it would ensure each is built & linked in the correct order every time. Something like this maybe

```json
"build": "ng build --prod && node scripts/link-packages @clr/angular"
```

Reply:
I thought about this and in the end, when you want to publish the package, do you really want the build to run `link`?

Reply:
Ideally, no, but I think it's helpful when the build is consistent as possible. Link might add a bit of extra work, but it simplifies the workflow and reduces the chance that the build output is inconsistent. If we want to publish, see if a local branch will pass CI, or test for a production bug, it's always just run `build`.

The package build+link step for each project independently is an optimization that can be added later since this will help unblock a few other PRs.